js homework week2/file6.js

```diff
@@ -0,0 +1,7 @@
+let flowers = [];
+console.log('the value of my array will be: names of flowers');

```

Comment:
At this point in the code, the value of the array is not `names of flowers`.

Reply:
ok done.